File: Course_01/C++ Language/EstruturaRepetitiva/media_idades.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int idade(0), soma(0), contador(0);
    double media;

    cout << "Digite as idades:" << endl;

    while (idade >= 0) {
        cin >> idade;
        if (idade > 0) {
            contador++;
            soma += idade;
        }
    }

    if (contador > 0) {
        media = (double) soma / contador;
        cout << fixed << setprecision(2) << "MEDIA = " << media <<endl;
    }
    else {
        cout << "IMPOSSIVEL CALCULAR";
    }
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/media_idades.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class media_idades {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int idade, soma, contador;
		double media;
		
		System.out.println("Digite as idades: ");
		idade = sc.nextInt();
		
		soma = 0;
		contador = 0;
		
		while (idade > 0) {
			soma += idade;
			contador++;
			idade = sc.nextInt();
		}
		
		media = (double)soma / contador;
		
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		sc.close();
	}

}
